Practice2/README.md

```diff
@@ -48,10 +61,18 @@
       * testimonials.css
       * version.css
     * variables/
-      * main-var.css
+      * all.css
+      * breakpoints.css
+      * color-heading.css
+      * color-text.css
+      * color.css
```

Comment:
`color-heading`, `color-text`, and `color` are just colors. One file for them is enough.

Reply:
Fixed